docs/user_guide/models/destvi.rst

```diff
@@ -0,0 +1,206 @@
+
+    + Scalable to very large datasets (>1 million cells).
+
+The disadvantages of DestVI include:
+
+    + Effectively requires a GPU for fast inference.
+
+.. topic:: Tutorial:
+
+ - :doc:`/tutorials/notebooks/DestVI_tutorial`
+
+
+Preliminaries
+=============
+DestVI requires training two models, the scLVM (single-cell latent variable model) and the
+stLVM (spatial transcriptomic latent variable model). The scLVM takes in as input a scRNA-seq gene
+expression matrix of UMI counts :math:`X` with :math:`N` cells and :math:`G` genes, along with
+a vector of cell type labels :math:`\vec{c}`. Subsequently, the stLVM takes in the trained scLVM
+model, along a spatial gene expression matrix :math:`Y` with :math:`S` spots and :math:`G` genes.
+Optionally, the user can specify the number of components used for the mixture model underlying the
+emprical prior.
+
+
+Generative process
+==================
+
+scLVM
+-----
+
+For cell :math:`n`, the scLVM assumes observed discrete cell type labels :math:`c_n` and models
+continuous covariates :math:`\gamma_n` of dimension :math:`d` to explain variation in gene expression within a cell type.
+The scLVM posits that the observed UMI counts for cell :math:`n` are generated by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        \gamma_n &\sim \textrm{Normal}(0, I) \tag{1} \\
+        x_{ng} &\sim \textrm{NegativeBinomial}(l_nf^g(c_n, \gamma_n), p_g) \tag{2} \\
+    \end{align}
+
+where :math:`l_n` is the library size, :math:`f` is a two-layer neural network which outputs a :math:`G`
+dimensional vector, and :math:`p_g` is the rate parameter of the negative binomial distribution for
+a given gene :math:`g`. Note that we are using the standard rate-shape parametrization of the
+negative binomial here, rather than the mean-dispersian parametrization used in :doc:`/user_guide/models/scvi`.
+
+The latent variables for the scLVM, along with their description are summarized in the following table:
+
+.. list-table::
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`\gamma_n \in \mathbb{R}^d`
+     - Low-dimensional representation of sub-cell-type covariates.
+     - ``z``
+   * - :math:`p_g \in \mathbb{R}^{+}`
+     - Rate parameter for negative binomial distribution.
+     - ``px_r``
+
+stLVM
+-----
+
+For the stLVM, we also model the expression counts with a :math:`\mathrm{NegativeBinomial}`. However,
+for spatial data, we assume that each spot :math:`s` has expression :math:`x_s` composed of gene expression
+counts of :math:`C(s)` individual cells. We assume that cell :math:`n` in spot :math:`s` is generated from
+the latent variables :math:`(c_{ns}, \gamma_{ns})` by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{n=1}^{C(s)}f^g(c_{ns}, \gamma_{ns}), p_g) \tag{3} \\
+    \end{align}
```

Comment:
I would not include this part and just stick with the one below, ignoring how this is derived.

Reply:
yes this is exact! Let's do this!